mod/stats.php

```diff
@@ -1,10 +1,14 @@
 <?php
+
 use Friendica\Directory\Rendering\View;
 
-if(! function_exists('stats_content')) {
-    function stats_content(&$a) {
-        $view = new View('stats');
-        $view->output();
-    }
+if (!function_exists('stats_content')) {
+	function stats_content(&$a)
+	{
```

Comment:
It's possible, this is my IDE formatting setup, I must have had a good reason to set it up like that, or it was like that by default. I'll revert to the most common way across friendica's repos.